02_activities/assignments/assignment.sh

```diff
@@ -1,22 +1,22 @@
 #!/bin/bash
  set -x
-m
+
 ################
 # Instructions #
 ################
 # > Edit this file below to complete the homework assignment
 # > Add your code below each comment to complete the tasks
 
 # 1. Create 5 directories named dir1, dir2, dir3, dir4, and dir5
- mkdir dir1, mkdir dir2, mkdir dir3, mkdir dir4, mkdir dir5
+ mkdir dir1 ... dir5
 # 2. List the contents of the parent directory to verify the presence of the 5 directories
```

Comment:
The comment I made previously regarding dir1 ... dir5 is a simplified version of the code. Please write out the full code.